src/index.js

```diff
@@ -159,6 +159,7 @@ export { JsonHandler } from './resources/json.js';
 export { JsonStandardMaterialParser } from './resources/parser/material/json-standard-material.js';
 export { KtxParser } from './resources/parser/texture/ktx.js';
 export { LegacyDdsParser } from './resources/parser/texture/legacy-dds.js';
+export { HdrParser } from './resources/parser/texture/hdr.js';
```

Comment:
Is it necessary to export this class to the `pc` namespace? (I know we already export the other parsers - just wondering...)

Reply:
I was wondering the same thing, I believe not. (I was surprised to see the others exported here).

Users can get to built-in parser instances via `app.loader.getHandler(pc.ASSET_TEXTURE).parsers.hdr`.

I vote to remove all the texture parsers from being exported?